src/db/migrations/20181004213420-createUsers.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
Basic question from my end. What does this file do for us?

Reply:
These file are database migrations. They contain mutations that are executed against our database. This one defines the Users table, along with its initial columns.